api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java

```diff
@@ -97,6 +106,13 @@
     @Inject
     private WhitelistEntryDataService whitelistEntryDataService;
 
+    @Inject
+    private CallDetailRecordDataService callDetailRecordDataService;
+
+    public UserControllerBundleIT() {
+        System.setProperty("org.motechproject.testing.osgi.http.numTries", "1");
```

Comment:
Should this also have the wait time property?

System.setProperty("org.motechproject.testing.osgi.http.msWait", "1000");